binder/environment.yml

```diff
@@ -0,0 +1,12 @@
+name: ipydatagrid
+channels:
+- conda-forge
+dependencies:
+- ipywidgets=7.6
+- jupyterlab=3
+- numpy
+- nodejs
+- pandas
+- pip
+- pip:
+  - ..
```

Comment:
Why having a pip section?

Reply:
Installs the package as is from repo?

Reply:
It was not available on conda forge when the PR was submitted. But maybe it is now.

Reply:
Also this is better because what you have in the repo is what you're using, so demos in repo will work even if main hasn't been released yet.

Reply:
I think we should rather use the same approach as in other widget packages where we make the binder point to a `stable` branch pointing to the latest release commit.

This prevents too frequent rebuilds of the binder, and also prevent people from facing issues with the potentially instable master version.

Reply:
I'd hope we don't land unstable PRs, but otherwise🤷‍♂️